clang/lib/Sema/SemaChecking.cpp

```diff
@@ -2920,6 +2920,24 @@ Sema::CheckBuiltinFunctionCall(FunctionDecl *FDecl, unsigned BuiltinID,
       return ExprError();
     break;
 
+  case Builtin::BI__builtin_cheriot_sealing_type: {
+    if (checkArgCount(TheCall, 1))
+      return ExprError();
+    auto *Arg = dyn_cast<StringLiteral>(TheCall->getArg(0));
+    if (!Arg)
+      return ExprError();
+    auto ArgLit = Arg->getString();
+    if (!isValidAsciiIdentifier(ArgLit)) {
+      std::string Escaped;
+      llvm::raw_string_ostream OS(Escaped);
+      llvm::printEscapedString(ArgLit, OS);
```

Comment:
Remove debugging code

Reply:
Maybe I'm missing something: it's not printing to stdout, it prints the escaped string to `std::string Escaped` above, which is then used in the message below

Reply:
Ah, I misread it.